conans/client/conf/__init__.py

```diff
@@ -13,12 +13,12 @@
 default_settings_yml = """
 # Only for cross building, 'os_build/arch_build' is the system that runs Conan
 os_build: [Windows, WindowsStore, Linux, Macos, FreeBSD, SunOS]
-arch_build: [x86, x86_64, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr]
+arch_build: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, sh4]
 
 # Only for building cross compilation tools, 'os_target/arch_target' is the system for
 # which the tools generate code
-os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, Arduino]
-arch_target: [x86, x86_64, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr]
+os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, Arduino, Neutrino]
+arch_target: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, sh4]
```

Comment:
Are those architectures closely related to QNX Neutrino? Maybe could go to another PR?

Reply:
- If ``ppcbe`` is implictly 32 bits, I am ok if the name doesn't contain 32.
- Why the arch is not called ``shle`` and called ``sh4``? Just asking, to understand.

Reply:
then I support better to name it `sh4le` (as there are SH1, SH2, SH3 and SH5, for example, so name just shle would be confusing)
we already have `ppc32` which is not just `ppc`, so probably it's more convenient to name it `ppc32be` then?

Reply:
Ok, good!

Reply:
I've renamed sh4 -> sh4le